bigtable/docs/snippets.py

```diff
@@ -425,5 +425,101 @@ def test_bigtable_set_iam_policy_then_get_iam_policy():
     assert len(policy.bigtable_admins) > 0
 
 
+def test_bigtable_admins_policy():
+    # [START bigtable_admins_policy]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable.policy import Policy
+    from google.cloud.bigtable.policy import BIGTABLE_ADMIN_ROLE
+
```

Comment:
Seems odd to include a blank line in the sample. Is this processed out? This is one of many instances of this formatting so the fix will need to be applied to other lines as well.